src/js/utils/rxjsUtils.js

```diff
@@ -0,0 +1,18 @@
+import { timer } from "rxjs/observable/timer";
+
+import "rxjs/add/operator/delayWhen";
+import "rxjs/add/operator/scan";
+
+/* eslint-disable import/prefer-default-export */
+export function linearBackoff(maxRetries = 5, delay = 1000, scheduler = null) {
+  return errors =>
+    errors
+      .scan((count, error) => {
+        if (count >= maxRetries) {
+          throw error;
+        }
+
+        return count + 1;
+      }, 0)
+      .delayWhen(val => timer(val * delay, null, scheduler));
```

Comment:
I like it that you make it flexible by passing the scheduler. But since it looks like you are not using it for tests, nor there is a test who covers this behavior, I would rather not have the linearBackoff care about it.

Other option would be to test passing a custom scheduler.

Reply:
I agree that it looks like a "just in case" passage, but I wanted to be a good RxJS citizen 👮